### PR TITLE
Removes the DNA scanner board from the research list and tech storage

### PR DESCRIPTION
**Pull request by Ezuo**
_They removed genetics, they said. People wouldn't be able to use it, they said._

A PR made out of pure spite and aggravation, this removes the station's ability to get the genetics DNA scanner/changer computers. This includes derelict diving.

The system still exists (against my better judgement), so it can be used for buse and a machine still exists on CC.

### CODE REVIEW

**Comment by HeyBanditoz** [comment]
Squash please.

**Comment by skull132** [comment]
@HeyBanditoz no need. We use Github's squash-and-merge. The end result is
the same, with less hassle for the developer.

**Comment by NonQueueingMatt** [comment]
I completely agree with removing the ability for the station to get genetics. It's a really aggravating new fad that scientists tunnelvision genetics and give themselves xray/telepathy. And while before almost nobody knew about the genetics part, now that everybody knows, genetics is being built every single round.

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 1]
```diff
@@ -33115,7 +33115,6 @@
 	},
 /obj/item/weapon/circuitboard/clonescanner,
 /obj/item/weapon/circuitboard/clonepod,
-/obj/item/weapon/circuitboard/scan_consolenew,
```
Comment (Lohikar) [inline maps/aurora/aurora-4_mainlevel.dmm]: Isn't this needed for cloning?

File: maps/aurora/aurora-8_derelict.dmm [hunk 2]
```diff
@@ -4334,7 +4334,6 @@
 /turf/simulated/floor/tiled/white,
 /area/derelict/hallway/southeast)
 "nt" = (
-/obj/machinery/computer/scan_consolenew,
```
Comment (Lohikar) [inline maps/aurora/aurora-8_derelict.dmm]: Same here.

**Comment by Ezuo** [comment]
As said in discord, thats not the cloning console. Thats the genetics DNA console.

File: maps/aurora/aurora-4_mainlevel.dmm (hunk 1 above)
Comment (TheGreatJorge) [reply maps/aurora/aurora-4_mainlevel.dmm]: Doesn't seem so. `/clonescanner`, `/clonepod` and `/cloning` seem to be boards related to cloning. `/scan_consolenew` is labeled as DNA Machine.

**Comment by Pacmandevil** [comment]
Protohumans have some actual valid uses. I feel it should just be made clear that you shouldn't be giving yourself or anyone who hasn't done absurd amount of paperwork for testing clearance superpowers.

Might be worth bringing it up with CCIA to get this IR-able.

File: maps/aurora/aurora-4_mainlevel.dmm (hunk 1 above)
Comment (skull132) [reply maps/aurora/aurora-4_mainlevel.dmm]: It is!

Comment (skull132) [reply maps/aurora/aurora-4_mainlevel.dmm]: Actually, no it isn't!

DNA machinery is fucking retarded in that the machine itself is called `dna_scannernew`, but the circuit inside of it is called `clonescanner`. Like, _What_.

File: maps/aurora/aurora-8_derelict.dmm (hunk 2 above)
Comment (skull132) [reply maps/aurora/aurora-8_derelict.dmm]: This is actually a thing on the derelict that should stay.

File: code/modules/research/designs/circuit_designs.dm [hunk 3]
```diff
@@ -79,11 +79,11 @@
 	build_path = /obj/item/weapon/circuitboard/pandemic
 	sort_string = "FAEAA"
 
-/datum/design/circuit/scan_console
+/*/datum/design/circuit/scan_console
```
Comment (skull132) [inline code/modules/research/designs/circuit_designs.dm]: Delete code instead of commenting it out, please and thank you.

**Comment by Arrow768** [comment]
Yup, that needs to be removed. There are certain scientists that build that almost every round.

**Comment by Pacmandevil** [comment]
surely they can be delt with using IRs/ahelping it? Removing something just because someone's abused it seems a bit harsh.

**Comment by Lohikar** [comment]
PR inactive for over 2 weeks. Closing until the author responds.

**Pull request closed without merging**